doc/newsboat.asciidoc

```diff
@@ -961,15 +961,18 @@ is <<ignore-article,`ignore-article`>>:
 
 	ignore-article "*" "title =~ \"Gentoo\""
 	ignore-article "https://synflood.at/blog/index.php?/feeds/index.rss2" "title =~ \"newsboat\""
-
-It takes two parameters. The first one is either a URL of a feed, or `"*"` to
-match any feed (asterisk is _not_ a pattern, glob or regex—we simply reserve it
-to mean "all feeds").  The second argument is <<_filter_language,a filter
-expression>> for an article, probably <<_using_double_quotes,in double quotes
-to preserve spaces inside>>. If Newsboat hits an article in the specified RSS
-feed that matches the specified filter expression, then this article is ignored
-and never presented to the user. The configuration itself can contain as many
-<<ignore-article,`ignore-article`>> commands as desired.
+	ignore-article "regex:https?://nitter.net/.*" "title =~ \"^RT by\""
+
+It takes two parameters. The first one is either a URL of a feed, a pattern 
+prefixed with `regex:`, this pattern being in the POSIX extended reg format, 
```

Comment:
Nitpicks:
- it'd be nice to link to https://pubs.opengroup.org/onlinepubs/9699919799.2008edition/basedefs/V1_chap09.html for the details on this particular regex grammar.
- please do call it "regex" rather than "reg"; the latter is hard to search for.